src/axes/timeAxis.ts

```diff
@@ -394,7 +394,7 @@ export module Axes {
       let xTranslate = (this._tierLabelPositions[index] === "center" || config.step === 1) ? 0 : this.tickLabelPadding();
       let yTranslate = this.orientation() === "bottom" ?
           d3.sum(this._tierHeights.slice(0, index + 1)) - this.tickLabelPadding() :
-          this.height() - d3.sum(this._tierHeights.slice(0, index)) - this.tickLabelPadding();
+          this.height() - this.innerTickLength() - d3.sum(this._tierHeights.slice(0, index)) - this.tickLabelPadding();
```

Comment:
This sounds like a separate bug and the fix does not necessarily seem correct (Why not `_maxLabelTickLength()`?)  Let's make a separate pull request for this with the appropriate bugs?